docs/frames/default.md

```diff
@@ -0,0 +1,134 @@
+import { TOKENS as VUE, services as vue } from '@/app/vue'
+import { services as kuma } from '@/app/kuma'
+import { build, token } from '@/services/utils'
+import Kongponents from '@kong/kongponents'
+import CliEnv from '@/services/env/CliEnv'
+import i18nEnUs from '@/locales/en-us'
+
+const el = ref()
+const $ = {
+  ...VUE,
+  ...APP,
+  globals: token('vue.globals'),
+}
+
+onMounted(async () => {
+  createApp(el.value, {
+    enhanceApp: (app) => {
+      const get = build(
+        vue($),
+        application($),
+        kuma($),
+        [
+          // temporary $.app replacement
+          [$.app, {
+            service: (
+              components,
+              plugins,
+              globals,
+            ) => {
+              return async (app) => {
+                components.forEach(([name, component]) => {
+                  app.component(name, component)
+                })
+
+                plugins.forEach(([...args]) => {
+                  app.use(...args)
+                })
+
+                globals.forEach(([name, obj]) => {
+                  app.config.globalProperties[name] = obj
+                })
+
+                return app
+              }
+            },
+            arguments: [
+              $.components,
+              $.plugins,
+              $.globals,
+            ],
+          }],
+          [token('application.routes.navigation.guards'), {
+            service: () => {
+              return []
+            },
+            labels: [
+              $.routesLabel,
+            ],
+          }],
+
+          [token('kong.plugins'), {
+            service: () => {
+              return [
+                [Kongponents],
+              ]
+            },
+            labels: [
+              $.plugins,
+            ],
+          }],
+          [token('docs.globals'), {
+            service: (i18n) => {
+              return [
+                ['t', i18n.t],
+              ]
+            },
+            arguments: [
+              $.i18n
+            ],
+            labels: [
+              $.globals,
+            ],
+          }],
+          [$.Env, {
+            service: CliEnv,
+            arguments: [
+              $.EnvVars,
+            ],
+          }],
+
+          [$.EnvVars, {
+            constant: {
+              KUMA_PRODUCT_NAME: '',
+              KUMA_FEEDBACK_URL: '',
+              KUMA_CHAT_URL: '',
+              KUMA_INSTALL_URL: '',
+              KUMA_VERSION_URL: '',
+              KUMA_DOCS_URL: '',
+              KUMA_MOCK_API_ENABLED: '',
+              KUMA_ZONE_CREATION_FLOW: '',
+            },
+          }],
+          [token('kuma.i18n.en-us'), {
+            constant: i18nEnUs,
+            labels: [
+              $.enUs,
+            ],
+          }],
+        ],
+      )
+      get($.app)(app)
+    }
+  })
+})
+</script>
+
+<div id="sandboxed-component" ref="el"></div>
+
+<style scoped>
+#sandboxed-component {
+  width: 100%;
+  height: 100vh;
+  padding: 0.5rem;
```

Comment:
**Suggestion**: To ensure this doesn’t look weird in dark mode.

```suggestion
  padding: 0.5rem;
  background-color: #fff;
```

![image](https://github.com/kumahq/kuma-gui/assets/5774638/42ac7bcd-eb86-4225-93c0-1e37bc9d3acf)

Reply:
Resolving this following https://github.com/kumahq/kuma-gui/pull/1807#issuecomment-1828302119.